### PR TITLE
Fix compilation on Flutter 3.13.0 stable

### PR DESCRIPTION
**Pull request by h3x4d3c1m4l**
This PR (rudimentarily) fixes compilation on Flutter 3.13.0 stable. I tested running the example on Windows, it runs and I haven't found anything broken yet.

However, as the breaking changed APIs were not documented ([here](https://docs.flutter.dev/release/breaking-changes#released-in-flutter-313) or [here](https://docs.flutter.dev/release/breaking-changes/3-10-deprecations)) and knowing my knowledge of both Fluent UI's and Flutter's internals is limited, these changes might not be the correct...

## Pre-launch Checklist

- [X] I have updated `CHANGELOG.md` with my changes  
- [X] I have run "dart format ." on the project  
- [X] I have added/updated relevant documentation

### CODE REVIEW

**Comment by h3x4d3c1m4l** [comment]
Ah, analysis fails due to [ignoringSemantics](https://api.flutter.dev/flutter/widgets/IgnorePointer/ignoringSemantics.html) being deprecated. However I can't seem to find the equivalent of the current implementation...

I don't think `ExcludeSemantics` widget will do as it will cause the opposite effect, right?

**Review by ricardoboss — APPROVED** [review]

File: lib/src/controls/form/selection_controls.dart [hunk 1]
```diff
@@ -236,13 +233,10 @@ class _FluentTextSelectionControlsToolbarState
...
       if (_clipboardStatus != null) {
         _clipboardStatus!.removeListener(_onChangedClipboardStatus);
-        _clipboardStatus!.dispose();
       }
       _clipboardStatus = widget.clipboardStatus;
       _clipboardStatus!.addListener(_onChangedClipboardStatus);
-      if (widget.handlePaste != null) {
-        _clipboardStatus!.update();
-      }
+      if (widget.handlePaste != null) {}
```
Comment (bdlukaa) [inline lib/src/controls/form/selection_controls.dart:239]: This does?

Comment (h3x4d3c1m4l) [reply lib/src/controls/form/selection_controls.dart:239]: Absolutely nothing 😂

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 * fix: Render `Combobox`'s elevation outside of the clipper ([#896](https://github.com/bdlukaa/fluent_ui/discussions/896))
 * fix: Do not make items exclusive on `NavigationView`'s body ([#913](https://github.com/bdlukaa/fluent_ui/issues/913))
 * fix: `Expander.content` has now its focus excluded when closed
+* fix: Fixed compile errors with Flutter 3.13.0 stable
```
[the remark this answers is not in the document]
Comment (h3x4d3c1m4l) [reply CHANGELOG.md]: Fixed!

**Pull request merged**